mne/inverse_sparse/tests/test_sure.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from numpy.linalg import norm
+
+import os.path as op
+
+import mne
+from mne.inverse_sparse.mxne_inverse import mixed_norm
+from mne.viz import plot_sparse_source_estimates
+from mne.datasets import sample
+
+if __name__ == "__main__":
```

Comment:
no need for if main

Reply:
and see how tests are written elsewere with test_blah_blah to be used with pytest

Reply:
also I would rather add a test in test_mxne_inverse.py rather than create a new function